buildroot/share/PlatformIO/variants/MARLIN_G0B1RE/PeripheralPins.c

```diff
@@ -253,7 +252,6 @@ WEAK const PinMap PinMap_UART_RX[] = {
   {PC_11_ALT1, USART4,  STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF1_USART4)},
   {PD_2,       USART5,  STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF3_USART5)},
   {PD_6,       USART2,  STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF0_USART2)},
-  {PD_9,       USART3,  STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF4_USART3)},
```

Comment:
In the Arduino framework the `PinMap_UART_RX` table is used by the `HardwareSerial` constructor and the `digitalPinHasSerial` macro. And the STM32 framework uses it for `uart_init`. But it's the `HAVE_HWSERIAL#` macros (and/or `SERIAL_UART_INSTANCE`) that determine whether a port will be functional. So, should this line be included even though the serial port is not being activated elsewhere?

Reply:
@bigtreetech would have to answer that.